### ISSUE TITLE
cannot read property of leingth undifined is just one of many errors the latest vortex install. 

### ISSUE THREAD

**Issue by tetoNidan** [rockerbacon/lutris-skyrimse-installers#25]
Just reinstalled vrotex using your lutris runner lurtis -i /file.ymal file method.

Failed to initialize loot:

Error: Invalid parameter. 'C:/users/took/Local Settings/Application Data/Fallout4'
    at handleResponse (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\gamebryo-plugin-management\index.js:33:70723)
    at Socket.e.on.e (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\gamebryo-plugin-management\index.js:33:69360)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)

----------------------------------------------------------------------------------

lutris log output:

Running /home/took/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/wine /home/took/vortex/drive_c/Program Files/Black Tree Gaming Ltd/Vortex/Vortex.exe
Initial process has started with pid 22759
Game is considered started.

info: Epic games launcher not found error=File not found. (2) 'SOFTWARE\WOW6432Node\Epic Games\EpicGamesLauncher'

info: steam not found error=File not found. (2) 'Software\Valve\Steam'

info:  Game: Vortex.exe
info:  DXVK: v1.6.1+
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  GeForce GTX 1070:
info:    Driver: 440.82.0
info:    Vulkan: 1.1.119
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 8970 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Device properties:
info:    Device name:     : GeForce GTX 1070
info:    Driver version   : 440.82.0
info:  Enabled device extensions:
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_conditional_rendering

---------------------------------------------------------------------------------

Text popup not text copyable:

![Selection_018](https://user-images.githubusercontent.com/25040460/81127809-aaff9200-8efc-11ea-8d77-8fa28fb83b2b.png)

**Comment by rockerbacon** [comment]
When are these errors raised?

**Comment by tetoNidan** [comment]
Yesterday.

**Comment by rockerbacon** [comment]
I'm asking what actions do you take inside Vortex raise these errors? eg.: "Sometimes I deploy mods I get this error"

**Comment by tetoNidan** [comment]
Start it up, manually set fallout 4 folder because it is not found even after using tour symbolic link bash script. There was one more error box telling me that if I continue not to wine when thing get funked up. So I would just say this is general use of the software.

**Comment by rockerbacon** [comment]
Check if your application data directory for Fallout 4 exists in the Vortex prefix with this command:

```
ls "$HOME/Games/vortex-mod-manager/drive_c/users/$USER/Local Settings/Application Data/Fallout4"
```

If it doesn't, show me output for the command:

```
"$HOME/Games/vortex-mod-manager/config_scripts/vortex-symlinks.sh"
```

**Comment by rockerbacon** [comment]
The many problems are most probably caused by lutris/lutris#2871, seeing as the user solved their issues in #44 by fixing a bad Winetricks installation.

As the user got MO2 installed and probably has no reason to use Vortex anymore, I'll be closing this issue
